src/pages/FolderSharedPage.jsx

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useState } from "react";
+import "../styles/FolderSharedPage.css";
+import Footer from "../common/footer/Footer";
+import NavigationBar from "../common/navigationBar/NavigationBar";
+import Cards from "../components/cards/Cards";
+import SearchBar from "../common/searchBar/SearchBar";
+import UserInformation from "../components/folderInformation/FolderInformation";
+import { getSampleCards, getSampleUser } from "../services/api";
+
+function FolderSharedPage() {
+  const [folderOwners, setFolderOwners] = useState([]);
+  const [folderName, setFolderName] = useState();
+  const [SampleCards, setSampleCards] = useState([]);
```

Comment:
요기는 왜 state명이 대문자인가요 👀 소문자로 통일되면 좋겠어요!
```suggestion
  const [sampleCards, setSampleCards] = useState([]);
```